src/HtmlGenerator/Program.cs

```diff
@@ -1,4 +1,7 @@
-﻿using System;
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+

```

Comment:
Will do, sorry about that I accidentally applied the code formatter from dotnet/corefx